src/current/cockroachcloud/cloud-api.md

```diff
@@ -172,47 +184,59 @@ If the request was successful, the API will return information about the newly c
 {% include_cached copy-clipboard.html %}
```

Comment:
I think this is the response so it should not be copyable. I'm actually not even sure we need to show the response since the API spec has it already. WDYT?